release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1598,6 +1598,156 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, when you installed a cluster and tried to add a worker node with the `oc adm node-image` command it failed because the date, time, or both might have been inaccurate. With this release, the issue is resolved by applying the same Network Time Protocol (NTP) configuration in the target cluster `MachineConfig` chrony resource to the node ephemeral live environment (link:https://issues.redhat.com/browse/OCPBUGS-42544 [*OCPBUGS-42544*])
+
+* Previously, during installation the name of the artifact that the `oc adm node-image create` command generated did not include `<arch>` in its file name. As a consequence, the file name was inconsistent with other generated ISOs. With this release, a patch fixes the name of the artifact that is generated by the `oc adm node-image create` command by also including the referenced architecture as part of the file name and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-42528[*OCPBUGS-42528*])
+
+* Previously, the Agent-based Installer set the `assisted-service` object to a debug logging mode. Unintentionally, the `pprof` module in the `assisted-service` object was then turned on and this uses the port `6060`. As a consequence, there was a port conflict and the `cloud-credential-operator` did not run. When requested by the {vmw-first} `cloud-controller-manager`, {vmw-short} secrets were not generated, the OpenStack Cloud Controller Manager (CCM) failed to initialize the nodes, and the cluster installation was blocked. With this release, the `pprof` module in the `assisted-service` object does not run when invoked by the Agent-based Installer. As a result, the `cloud-credential-operator` runs correctly and cluster installations on {vmw-short} that use the Agent-based Installer succeed. (link:https://issues.redhat.com/browse/OCPBUGS-42525[*OCPBUGS-42525*])
+// --- page 4
+* Previously, when a worker node was trying to join a cluster the rendezvous node rebooted before the process completed. As the worker node could not communicate as expected with the rendezvous node, the installation was not successful. With this release, a patch is applied that fixes the racing condition that caused the rendezvous node to reboot prematurely and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-41811 [*OCPBUGS-41811*])
+
+* Previously, when using the {ai-full}, selecting a multiarch image for `s390x` CPU architecture on {hybrid-console} could cause the installation to fail. The installation program reported an error that the new cluster was not created because the skip MCO reboot was not compatible with `s390x` CPU architecture. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41716[*OCPBUGS-41716*])
```

Comment:
```suggestion
* Previously, when using the {ai-full}, selecting a multi-architecture image for `s390x` CPU architecture on {hybrid-console} could cause the installation to fail. The installation program reported an error that the new cluster was not created because the skip MCO reboot was not compatible with `s390x` CPU architecture. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41716[*OCPBUGS-41716*])
```
In portal searches, I saw the full compound word - for your consideration.